draft-ietf-quic-manageability.md

```diff
@@ -945,37 +945,59 @@ severe packet loss.
+compliance with ciphersuites); client and application fingerprinting for
+inventory; and to provide alerts for network intrusion detection and other
+next generation firewall functions.
+
 Current practices in detection and mitigation of Distributed Denial of Service
 (DDoS) attacks generally involve classification of incoming traffic (as
 packets, flows, or some other aggregate) into "good" (productive) and "bad"
 (DDoS) traffic, and then differential treatment of this traffic to forward only
 good traffic. This operation is often done in a separate specialized mitigation
 environment through which all traffic is filtered; a generalized architecture
 for separation of concerns in mitigation is given in
- {{?DOTS-ARCH=I-D.ietf-dots-architecture}}.
+{{?DOTS-ARCH=I-D.ietf-dots-architecture}}.
 
-Key to successful DDoS mitigation is efficient classification of this traffic in
-the mitigation environment.  Limited first-packet garbage detection as in
-{{sec-garbage}} and stateful tracking of QUIC traffic as in {{sec-stateful}}
-above may be useful during classification.
+Efficient classification of this DDoS traffic in the mitigation environment
+is key to the success of this approach. Limited first-packet garbage detection
+as in {{sec-garbage}} and stateful tracking of QUIC traffic as in
+{{sec-stateful}} above may be useful during classification.
 
 Note that the use of a connection ID to support connection migration renders
 5-tuple based filtering insufficient and requires more state to be maintained by
-DDoS defense systems. For the common case of NAT rebinding, DDoS defense systems
+DDoS defense systems. For the common case of NAT rebinding, where the client's
+address changes without the client's intent or knowedge, DDoS defense systems
 can detect a change in the client's endpoint address by linking flows based on
-the server's connection IDs. QUIC's linkability resistance ensures that
+the server's connection IDs. However, QUIC's linkability resistance ensures that
 a deliberate connection migration is accompanied by a change in the connection
-ID.
-
-It is questionable whether connection migrations must be supported during a DDoS
-attack. If the connection migration is not visible to the network that performs
-the DDoS detection, an active, migrated QUIC connection may be blocked by such a
-system under attack. As soon as the connection blocking is detected by the
-client, the client may rely on the fast resumption mechanism provided by
-QUIC. When clients migrate to a new path, they should be prepared for the
-migration to fail and attempt to reconnect quickly.
-
-TCP syncookies {{?RFC4937}} are a well-established method of mitigating some
+ID. In this case, the connection ID can not be used to distinguish valid, active
+traffic from new attacl traffic.
+
+Alternatively, when an encrypted transport is used, it is also possible for
+endpoints to directly support these security functions.
```

Comment:
I don't quite understand this sentence. For one, why "when an encrypted transport is used"? This document is purely about an encrypted transport, right? And then "these security functions" is also unclear to me.

Reply:
On the first: I think this is true for any encrypted transport, but yes it does apply to QUIC.  
On the functions: I thought this was: DOS;DDOS;Malware detection; NIDS; etc